taglib/mp4/mp4atom.cpp

```diff
@@ -32,11 +32,13 @@
 
 using namespace TagLib;
 
-const char *const MP4::Atom::containers[11] = {
+namespace {
+  constexpr std::array<const char *, 11> containers {
```

Comment:
I think the template arguments could be removed here now that the declaration is no longer in the header.

Reply:
Done.